react_app/src/App.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import EmbeddingForm from './Form';
 import styles from './App.scss';
+import {ThreeRenderer} from './threeVisualizer.js'
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { datasets: [] };
+    this.state = { datasets: [],
+                    selectedDataset: "" };
```

Comment:
single quotes, here and in all the files.

Reply:
Fixed!